packages/replay/src/bin.ts

```diff
@@ -109,7 +109,7 @@ commandWithGlobalOptions("rm-all")
 
 commandWithGlobalOptions("update-browsers")
   .description("Update browsers used in automation.")
-  .arguments("<browsers...>")
+  .arguments("[<browsers...>]")
```

Comment:
This is a valid change but please don't add put unrelated changes in the same PRs